queries.sql

```diff
@@ -1,5 +1,3 @@
-/*Queries that provide answers to the questions from all projects.*/
-
 SELECT * FROM animals WHERE name LIKE '%mon%'
```

Comment:
- [ ] It would be better if we implement the query to find all animals that its name ends in "mon". Currently, our query is finding all animals that `begins and ends` in "mon"
**Hint**: `'%mon'`